base/tests.py

```diff
@@ -219,7 +219,7 @@ class TestGeo(TestCase):
                                       'southwest': {'lat': -0.7461185, 'lng': 8.753189299999999}},
                            'viewport': {'northeast': {'lat': -0.7244695, 'lng': 8.767470699999999},
                                         'southwest': {'lat': -0.7461185, 'lng': 8.753189299999999}},
-                           'location': {'lat': -0.7345885999999999, 'lng': 8.760486199999999},
```

Comment:
Probabilmente conviene fare il test su altri elementi più costanti.
Un test del genere è comunque abbastanza fragile perché dipende da come google aggiorna le sue mappe, però visto lo scopo del test temo sia inevitabile mettere in pianificazione che richiederà un po' di manutenzione